source/includes/api/_query.md

```diff
@@ -312,6 +333,27 @@ Query parameter        | Description
 sql                    | The SQL query to be executed. This parameter changes the data returned in the query response body. | String | Yes |
 format                 | The format of the returned response. By default, CSV format is assumed (`csv`), but you can also request the response as JSON (`json`). Check the section on the [download endpoint response body](index-rw.html#download-response-body) for some examples of how the `format` query parameter can be used. | String | No |
 freeze                 | The `freeze` parameter, when provided as `true`, will create a file with the results of the execution of the query and return the URL for that file. **Please note that you should be authenticated in order to request freezing the results of query executions.** | Boolean | No |
+geostore               | Read more about the geostore query parameter [here](/index-rw.html#filter-download-results-by-geostore). | String | No |
+
+#### Filter download results by geostore
+
+> Example download request providing a geostore id as query parameter to filter the results:
+
+```shell
+curl -i -X GET 'http://api.resourcewatch.org/v1/query/1d7085f7-11c7-4eaf-a29a-5a4de57d010e?sql=SELECT * FROM dis_001_significant_earthquakes LIMIT 5&geostore=972c24e1da2c2baacc7572ee9501abdc'
```

Comment:
Should this be a download request? Looks like a normal query request to me...

Reply:
Yes, it should! Thanks for pointing out :)